host_modules/systemd_service.py

```diff
@@ -48,3 +54,19 @@ def stop_service(self, service):
         if result.returncode:
             msg = result.stderr.decode()
         return result.returncode, msg
+
+    @host_service.method(host_service.bus_name(MOD_NAME), in_signature='s', out_signature='is')
```

Comment:
Please update the `in_signature` to be an integer.

Reply:
Fixed it, Thank you!